atomics/T1217/T1217.yaml

```diff
@@ -133,3 +133,56 @@ atomic_tests:
     cleanup_command: |
       rm -f #{output_file} 2>/dev/null
     name: sh
+- name: Extract Edge Browsing History
```

Comment:
Hello @likhitha0541 Thanks for your contribution. Looks like there is a minor issue with your YAML file. Can you check this against one of the YAML validators like [this website](https://jsonformatter.org/yaml-validator) to format and validate whether the YAML is valid ? Thanks in advance.